pixivpy3/api.py

```diff
@@ -42,15 +42,7 @@ def set_accept_language(self, language):
 
     def parse_json(self, json_str):
         """parse str into JsonDict"""
-
-        def _obj_hook(pairs):
-            """convert json object to python object"""
-            o = JsonDict()
-            for k, v in pairs.items():
-                o[str(k)] = v
-            return o
-
-        return json.loads(json_str, object_hook=_obj_hook)
+        return json.loads(json_str, object_hook=lambda obj: JsonDict(obj))
```

Comment:
It could be simplified to `return json.loads(json_str, object_hook=JsonDict)`

Reply:
Thanks for reminding
First time to use object_hook